client/blocks/login/index.jsx

```diff
@@ -796,6 +840,8 @@ class Login extends Component {
 
 				{ this.renderNotice() }
 
+				{ this.renderToS() }
```

Comment:
Shouldn't it be placed after the footer?
here:
<img width="379" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/e6e84ea6-a5b7-4f77-ba2d-192655a290e0">

instead of
<img width="469" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/106bddeb-8142-40d8-9c14-7acabce7d7ed">

Reply:
Design is under the header 👍 
4lWBkrFl1jQuN6irMuoLys-fi-306%3A1268

![image](https://github.com/Automattic/wp-calypso/assets/52076348/4fb28ebd-def3-4622-9476-fb5fbf735206)

(You have to look at the third one)